src/main/java/uk/gov/hmcts/darts/event/service/impl/CleanupCurrentFlagEventProcessorImpl.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.darts.event.service.impl;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Pageable;
+import uk.gov.hmcts.darts.common.repository.EventRepository;
+import uk.gov.hmcts.darts.event.service.CleanupCurrentFlagEventProcessor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Getter
+@RequiredArgsConstructor
+@Slf4j
+public class CleanupCurrentFlagEventProcessorImpl implements CleanupCurrentFlagEventProcessor {
+    private final Integer batchSize;
+    private final EventRepository eventRepository;
+
+    @Override
+    public List<Integer> processCurrentEvent() {
+        List<Integer> processedEventIdLst = new ArrayList<>();
+        log.info("Batch size to process event ids for {}", batchSize);
```

Comment:
is all this INFO logging really needed? I'm thinking that it could produce too much noise in production. Perhaps we can set some of these to DEBUG?

Reply:
I will change to debug for now. Thanks